Makefile

```diff
@@ -28,6 +28,7 @@ JAILER_BIN=$(FC_TEST_DATA_PATH)/jailer-main
 UID = $(shell id -u)
 GID = $(shell id -g)
 
+# Version has to be in format of vx.x.x
 firecracker_version=v1.4.0
```

Comment:
Do you want to move directly to 1.4.1 https://github.com/firecracker-microvm/firecracker/releases/tag/v1.4.1?

Reply:
Sure.